### PR TITLE
fixes# 28168 - added registered users template

### PR DESCRIPTION
**Pull request by domitea**

### CODE REVIEW

File: report_templates/registered_users.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<%#
+name: List of registered users
```
Comment (ares) [inline report_templates/registered_users.erb]: I'd call it just `Registered users` to remain consistent with other reports. Reports are lists usually, so we don't need to repeat that information

**Review by ares — CHANGES_REQUESTED** [review]
Looks good to me, needs https://github.com/theforeman/foreman/pull/7155 to be in first in order to test. I left just one nit pick.

File: report_templates/registered_users.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  input_type: user
+  description: Limit the report only on users found by this search query. Keep empty
+    for report on all available users.
+  advanced: false
+  value_type: search
+  resource_type: User
+model: ReportTemplate
+-%>
+<%- load_users(search: input('Users filter'), includes: :auth_source).each_record do |user| -%>
+  <%-  report_row({
```
Comment (ares) [inline report_templates/registered_users.erb]: the indentation here is incorrect, it should start on col 0 and the indentation should be done inside of the ERB, see other reports for an example

File: report_templates/registered_users.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  description: Limit the report only on users found by this search query. Keep empty
+    for report on all available users.
+  advanced: false
+  value_type: search
+  resource_type: User
+model: ReportTemplate
+-%>
+<%- load_users(search: input('Users filter'), includes: :auth_source).each_record do |user| -%>
+<%-   report_row({
+         'Login': user.login,
```
Comment (ares) [inline report_templates/registered_users.erb]: sorry for nitpicking, but the indentation should be by 2 spaces, this is 3 spaces comparing to the line above

**Comment by domitea** [comment]
Is it right now? @ares

**Review by ares — APPROVED** [review]
Good, tested this with the linked PR and works great! Ready to merge when core PR gets in

**Comment by mmoll** [comment]
merged, díky @domitea!

**Pull request merged**